scss/civicrm/contact/pages/_export-contact.scss

```diff
@@ -36,10 +44,30 @@
     }
   }
 
-  .crm-submit-buttons {
-    > .crm-button {
-      margin-left: 0 !important;
-      margin-right: 10px !important;
+  .crm-form-block {
+    background-color: transparent;
+    box-shadow: none;
+
+    > .crm-submit-buttons {
+      background-color: #ffffff;
```

Comment:
variable

Reply:
Why we need it so say background white? isn't it white already?
Note that this component is already created for common solution. Why this needs a different solution then normal one? 
and if this solution is appropriate why not to apply at other places as well?